docs/configuration.rst

```diff
@@ -0,0 +1,556 @@
+tar_executable (default ``"pghoard_gnutaremu"``)
+  The tar command to use for restoring basebackups. This must be GNU tar because some
+  advanced switches like ``--transform`` are needed. If this value is not defined (or
+  is explicitly set to ``"pghoard_gnutaremu"``), Python's internal tarfile
+  implementation is used. The Python implementation is somewhat slower than the
+  actual tar command and in environments with fast disk IO (compared to available CPU
+  capacity) it is recommended to set this to ``"tar"``.
+restore_prefetch (default ``transfer.thread_count``)
+  Number of files to prefetch when performing archive recovery.  The default
+  is the number of Transfer Agent threads to try to utilize them all.
+
+
+.. _configuration_logging:
+
+Logging configuration
+~~~~~~~~~~~~~~~~~~~~~
+
+log_level (default ``"INFO"``)
+            Determines log level of ``pghoard``.
+syslog (default ``false``)
+  Enable / disable syslog logging
+syslog_address (default ``"/dev/log"``)
+  Determines syslog address to use in logging (requires syslog to be true as
+  well)
+syslog_facility (default ``"local2"``)
+  Determines syslog log facility. (requires syslog to be true as well)
+
+
+.. _configuration_monitoring:
+
+Monitoring
+~~~~~~~~~~
+
+alert_file_dir (default ``backup_location`` if set else ``os.getcwd()``)
+  Directory in which alert files for replication warning and failover are
+  created.
+stats (default ``null``)
+  When set, enables sending to a statsd daemon that supports Telegrag or DataDog
+  syntax with tags.
+  The value is a JSON object, for example::
+
+    {
+        "host": "<statsd address>",
+        "port": <statsd port>,
+        "format": "<statsd message format>",
+        "tags": {
+            "<tag>": "<value>"
+        }
+    }
+
+  host
+    The statsd host address
+  port
+    The statsd listening port
+  format (default ``"telegraf"``)
+      Determines statsd message format. Following formats are supported:
+
+        - ``telegraf`` `Telegraf spec <https://github.com/influxdata/telegraf/tree/master/plugins/inputs/statsd>`_
+        - ``datadog`` `DataDog spec <http://docs.datadoghq.com/guides/dogstatsd/#datagram-format>`_
+
+  :tags: (default null)
+      The tag key can be used to enter optional tag values for the metrics
+push_gateway (default ``null``)
+  When set, enables sending metrics to a Prometheus Pushgateway with tags.
+  The value is a JSON obejct, for example::
+
+    {
+        "endpoint": "<pushgateway address>",
+        "tags": {
+            "<tag>": "<value>"
+        }
+    }
+
+  endpoint
+    The pushgateway address
+  tags
+    An object mapping tags to their values.
+
+
+.. _configuration_http:
+
+HTTP Server configuration
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The pghoard daemon needs to listen on an HTTP port for the archive command and
+for fetching of basebackups/WAL's when restoring if not using an object store.
+FIXME: if it's not required, can we disable it ?
```

Comment:
FIXME in the doc makes sense for missing doc, but FIXME for a missing feature seem a bit more strange.